src/components/synonyms/agSynonymTable.vue

```diff
@@ -270,9 +271,9 @@ export default {
      * work well here.
      */
     save: async function() {
-      if (!this.isAuthenticated) {
+      if (!this.editable) {
```

Comment:
this is minor, but the save button will never be present when there isn't an authenticated user so I don't think there would ever be a case this could be true.